lldb/test/API/lang/swift/clangimporter/Werror/TestSwiftStripWerror.py

```diff
@@ -26,17 +26,13 @@ def test(self):
 
         # Turn on logging.
         log = self.getBuildArtifact("types.log")
-        self.expect("log enable lldb types -f " + log)
+        self.expect('log enable lldb types -f "%s"' % log)
```

Comment:
we can use python f-strings now:
```suggestion
        self.expect(f'log enable lldb types -f "{log}"')
```

Reply:
Nice, I did not know about this feature!